File: app/src/main/java/net/ginteam/carmen/model/Service.java
package net.ginteam.carmen.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by vadik on 26.01.17.
 */

public class Service implements Serializable {

    @SerializedName("id")
    private int mId;

    @SerializedName("name")
    private String mName;

    @SerializedName("category_id")
    private int mCategoryId;

    public int getId() {
        return mId;
    }

    public String getName() {
        return mName;
    }

    public int getCategoryId() {
        return mCategoryId;
    }

}

File: app/src/main/kotlin/net/ginteam/carmen/kotlin/model/Service.kt
package net.ginteam.carmen.kotlin.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * Created by eugene_shcherbinock on 2/9/17.
 */
data class ServiceModel(val id: Int, val name: String,
                        @SerializedName("category_id") val categoryId : Int) : Serializable